### DISCUSSION TITLE
LoRA on StableLM

### DISCUSSION THREAD

**Discussion by aPaleBlueDot** [ml-explore/mlx#990] [Q&A] [answered]
Hi MLX community,
I'd like to apply lora.py on StableLM, yet I do not know which data format is required for the train/test/validate json files. Any pointers would be appreciated.
Cheers

**Comment by awni** [comment]
Check out the section in the [docs on the data formats](https://github.com/ml-explore/mlx-examples/blob/main/llms/mlx_lm/LORA.md#Data)

**Comment by aPaleBlueDot** [comment]
Thx! So the data format is independent of the ML model, I didn't know that.

**Reply by awni** [reply, marked as answer]
If you use the chat template then the tokenizer will handle applying the model specific template.

**Reply by aPaleBlueDot** [reply]
Great, so that's what took place behind the scenes when I ran the lora example provided.
Thanks again for your and your team's efforts on mlx.
